src/NSwag.CodeGeneration.CSharp/Templates/ClientTemplate.tt

```diff
@@ -86,7 +86,7 @@ public partial interface I<#=Model.Class#>
     }
 
 <#}#>
-    partial void PrepareRequest(System.Net.Http.HttpClient request, ref string url);
+    partial void PrepareRequest(System.Net.Http.HttpClient request, System.Text.StringBuilder urlBuilder);

```

Comment:
I think we shouldn't break this because the feature was available for more than a year... WrapSuccessResponses has been renamed two weeks after its introduction and an opt-in feature which makes this breaking change far less problematic.